web/client/components/style/vector/Manager.jsx

```diff
@@ -211,68 +206,7 @@ class Manager extends React.Component {
         this.props.onChangeStyle(newStyles);
     }
     changeSymbolType = (id, pointType) => {
-        // TODO FIX THIS
-
-        let defaultSymbolStyle = {};
```

Comment:
are you sure this is no longer needed?
also why you moved to annotations epic?

Reply:
@MV88 This logic has been moved to epic to trigger when annotations are opened to have a central place to set styles with defaults from plugin. Also I think async logic should be in epics whenever possible.